packages/e2e.test.foundry-sdk-generator/generateMockOntology.js

```diff
@@ -14,21 +14,29 @@
  * limitations under the License.
  */
 
+// @ts-check
```

Comment:
This file is the old test setup file from `foundry-sdk-generator` except now its run as a script. I've taken care to try to keep as many lines as they were to keep the diff down. Thus you'll see things like `setup` and `teardown` still being used below so that indentation stays the same.